benchmark/rclnodejs/client-stress-test.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2018 Intel Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/* eslint-disable camelcase */
+const rclnodejs = require('../../index.js');
+
+rclnodejs.init().then(() => {
+  console.log('The client will send a GetMap request(response contains a size of 10MB array) every 100ms in 1 hour.');
+  console.log('Beginning');
+
+  const node = rclnodejs.createNode('stress_client_rclnodejs');
+  const client = node.createClient('nav_msgs/srv/GetMap', 'get_map');
+  let count = 0;
+
+  let timer = setInterval(() => {
+    if (count > 72000) {
```

Comment:
72000 / 10 / 3600 = 2 hours? But it's 1 hour in the test desc.

Reply:
I will correct these problems, and add a README to demonstrate how to build and run the tests of Node.js/Python/C++, thanks!